server/pxf-service/src/main/java/org/greenplum/pxf/service/BridgeOutputBuilder.java

```diff
@@ -444,8 +446,15 @@ else if (field.type == DataType.TIMESTAMP.getOID())
                     else if (field.type == DataType.DATE.getOID())
                         return field.val.toString();
                     else
-                        return Utilities.toCsvText((String) field.val, QUOTE, true, true, true);
+                        return Utilities.toCsvText((String) field.val,
+                                QUOTE,
+                                QUOTE,
+                                NEWLINE,
+                                DELIMITER,
+                                true,
+                                true,
+                                true);
                 })
-                .collect(Collectors.joining(DELIMITER, "", NEWLINE));
+                .collect(Collectors.joining(DELIMITER + "", "", NEWLINE));
```

Comment:
Let's use String.valueOf(DELIMITER) or define a DELIMITER_STRING
```suggestion
                .collect(Collectors.joining(String.valueOf(DELIMITER), "", NEWLINE));
```